### DISCUSSION TITLE
I am using gino and want to integrate sqladmin, but due to the dependent sqlalchemy version, I cannot install it normally.

### DISCUSSION THREAD

**Discussion by vvanglro** [aminalaee/sqladmin#537] [Q&A] [answered]
This is pdm log:

![image](https://github.com/aminalaee/sqladmin/assets/43594924/027c3d43-1bb2-4269-8524-e89a55009a1a)

Is it possible to change the dependency to be greater than 1.3.

**Comment by aminalaee** [comment]
Hey @vvanglro 

I don't think that is possible, there were multiple changes after 1.4 and I think that's why Gino has pinned this version.
What I suggest is that if you can clone the project locally and loosen dependency in SQLAdmin to sqlalchemy >=1.3 and run tests to see what happens.

I will do the same when I get the time.

**Reply by vvanglro** [reply]
Yes, I think i agree with your statement, because to be compatible with 1.3 requires a lot of compatibility work in the current project code.
I have already tried: https://github.com/vvanglro/sqladmin/actions/runs/5421517479/jobs/9857048488

**Reply by aminalaee** [reply]
Ah right, Sqlalchemy asyncio support was introduced in 1.4 so that's not possible to loosen the dependencies.

**Reply by vvanglro** [reply, marked as answer]
oh. I think I found a way that can be used, I just need to create an independent sub-project in the existing project, and make sure that this sub-project can import the existing sqlalchemy model, and then should be able to manage through sqladmin?

**Comment by vvanglro** [comment]
I succeeded, as I said above, create a sub-project, and move the model created with gino into the sub-project environment, you can take over the use of .
